commands/set.md

```diff
@@ -13,7 +13,8 @@ The `SET` command supports a set of options that modify its behavior:
 * `NX` -- Only set the key if it does not already exist.
 * `XX` -- Only set the key if it already exists.
 * `KEEPTTL` -- Retain the time to live associated with the key.
-* `!GET` -- Return the old string stored at key, or nil if key did not exist. An error is returned and `SET` aborted if the value stored at key is not a string.
+* `GET` -- Return the old string stored at key, or nil if key did not exist. An error is returned and `SET` aborted if the value stored at key is not a string.
+* `IFEQ` -- Set the key if the given value matches the existing value. An error is returned and `SET` aborted if the value stored at key is not a string.
```

Comment:
Let's put IFEQ after NX and XX, because they are related.

Remove "An error is returned and `SET` aborted if the value stored at key is not a string.". It's not specific to the IFEQ option, is it?

Reply:
Sorry! I see now that "An error is returned and `SET` aborted if the value stored at key is not a string." is special code for IFEQ. Let's keep this.

Reply:
addressed @zuiderkwast